stl/inc/yvals_core.h

```diff
@@ -413,23 +420,26 @@
 #include <vcruntime.h>
 #include <xkeycheck.h> // The _HAS_CXX tags must be defined before including this.
 
-#define _STL_PRAGMA(PRAGMA)          _Pragma(#PRAGMA)
+// Note that _STL_PRAGMA is load-bearing;
+// it still needs to exist even once CUDA and ICC support _Pragma.
+#if defined(__CUDACC__) || defined(__INTEL_COMPILER)
+#define _STL_PRAGMA(PRAGMA) __pragma(#PRAGMA)

```

Comment:
Did you intend to replace the definition of `_STL_PRAGMA` on line 421? This redefinition with a different expansion makes the program ill-formed.

Reply:
Nevermind: apparently "Changes since my last review" flipped out and was showing me garbage.